src/microbe_stage/Microbe.Contact.cs

```diff
@@ -505,80 +505,88 @@ public void Kill()
             }
         }
 
+        // Queues either 1 corpse chunk or a factor of the hexes
         int chunksToSpawn = Math.Max(1, HexCount / Constants.CORPSE_CHUNK_DIVISOR);
 
         var chunkScene = SpawnHelpers.LoadChunkScene();
 
-        for (int i = 0; i < chunksToSpawn; ++i)
+        // An enumerator to step through all available organelles in a random order when making chunks
+        using (var organellesAvailableEnumerator = organelles.OrderBy(_ => random.Next()).GetEnumerator())
         {
```

Comment:
Adding a using loop marked more changes than I actually made- most is just indentation

Reply:
I switched this to a using statements, which we prefer in Thrive for exactly the reason of not needing to add extra indentation.